.github/workflows/bindings_ci.yml

```diff
@@ -16,8 +16,73 @@ env:
   CARGO_TERM_COLOR: always
   MATRIX_SDK_CRYPTO_NODEJS_PATH: bindings/matrix-sdk-crypto-nodejs
   MATRIX_SDK_CRYPTO_JS_PATH: bindings/matrix-sdk-crypto-js
+  # keep in sync with uniffi dependency in root Cargo.toml
+  UNIFFI_REV: fdb769b567865d9c5c7c682a18d0c1301a039c85
 
 jobs:
+  xtask-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Check xtask cache
+        uses: actions/cache@v3
+        id: xtask-cache
+        with:
+          path: target/debug/xtask
+          key: xtask-linux-${{ hashFiles('Cargo.toml', 'xtask/**') }}
+
+      - name: Install rust stable toolchain
+        if: steps.xtask-cache.outputs.cache-hit != 'true'
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Build
+        if: steps.xtask-cache.outputs.cache-hit != 'true'
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: -p xtask
+
+  test-codegen:
```

Comment:
Can you rename to `test-uniffi-codegen` or something like that? I believe it's important to know that's it's about UniFFI. Same for the job name (one line below).